scout/server/links.py

```diff
@@ -600,21 +603,12 @@ def cosmic_links(variant_obj):
     return cosmic_links
 
 
-def beacon_link(variant_obj, build=None):
+def beacon_link(variant_obj, build="37"):
```

Comment:
Hm, maybe add a comment or 😅 type hint to this one as most of the link builds are `int` already.

Reply:
I'm afraid it will break it, unless I add that it can be either an int or a str. Some cases have build as int and some as str, same old problem 🤔

Reply:
I can buy that, though we do have conversion fairly comprehensively now. Just pick the more common type for the default at least, and if it for some reason is different than the other link generations that use int as default comment why or add a conversion?

Reply:
Ah look, the build is set immediately before calling this function, so all good, I can set it as an int:

<img width="639" alt="image" src="https://user-images.githubusercontent.com/28093618/219607846-3e29197e-55be-47d4-90da-0f92bb5b9e37.png">

Reply:
Fixed!